platform/o.n.swing.laf.flatlaf/nbproject/project.xml

```diff
@@ -54,7 +54,8 @@
                     <build-prerequisite/>
                     <compile-dependency/>
                     <run-dependency>
-                        <specification-version>1.0</specification-version>
+                        <release-version>1</release-version>
```

Comment:
According to [module documentation](https://bits.netbeans.org/12.0/javadoc/org-openide-modules/org/openide/modules/doc-files/api.html#how-vers) this could be a range if you know this module works with previous as well as new incompatible versions.